src/main/groovy/val/Idator.groovy

```diff
@@ -95,9 +134,9 @@ class Idator implements Check {
      * @return The ResultMap output for evaluating the input using this Check
      */
     @Override
-    public ResultMap call(Object input, EvalContext ctx) {
-        if (toStash) ctx.stashed."${toStash}" = input //Stash input if requested
-        mCheck(input, ctx)
+    public ResultMap call(Object input) {
+        if (toStash) stashed."${toStash}" = input  //Stash input if requested
+        mCheck(traverser.traverse(input))
```

Comment:
the control should be flipped around to allow the traverser to shortcircuit the check